product_service_duration/models/product_template.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, fields, models
+import odoo.addons.decimal_precision as dp
+
+
+class ProductTemplate(models.Model):
+
+    _inherit = 'product.template'
+
+    service_user_id = fields.Many2one(
+        string='Service User',
+        comodel_name='res.users',
```

Comment:
This is not multi company friendly as it is. A company would only be selling services for their users.